shared/src/components/exercise-preview/index.cjsx

```diff
@@ -39,6 +39,17 @@ ExercisePreview = React.createClass
       content = _.compact([tag.name, tag.description]).join(' ') or tag.id
       isLO = _.include(['lo', 'aplo'], tag.type)
       {content, isLO}
+    sortTags: (tags, extractTag) ->
+      tags = _.sortBy(tags, 'name')
+      idTag = _.findWhere(tags, type: 'id')
+      loTag = _.find(tags, (tag) ->
+        {isLO} = extractTag(tag)
+        isLO
+      )
+      if idTag then tags.splice(_.indexOf(tags, idTag), 1)
+      if loTag then tags.splice(_.indexOf(tags, loTag), 0, idTag)
+      else tags.push(idTag)
+      tags
```

Comment:
I doubt it. it's not really the best named anyway. It's really an "insertSpecialTags" or something.